apps/showcase/src/styles.scss

```diff
@@ -34,31 +34,14 @@ h1, h2 {
 }
 
 .hljs-built_in, .hljs-symbol {
-  color: #bd5208;
+  color: #cb3544;
```

Comment:
can not be based on DF CSSvar?

Reply:
these are very specific colors used to highlight the keywords in code, so not sure DF variables are relevant for that
I can create a CSS variable for that though
(I had to change the color of the github theme a bit to adapt the contrast to the background that was a bit darker)